webpack.config.js

```diff
@@ -62,22 +62,6 @@ module.exports = [
     //   "react-router": "ReactRouter",
     //   "history": "History",
     // },
-  }),
-
-  Object.assign( {}, shared, {
-    entry: './specs.js',
-    devtool: 'eval',
-
-    target: 'node',
-
-    plugins: [
-      new webpack.IgnorePlugin( /ReactContext/ ),
-    ],
-
-    output: {
-      path: __dirname + '/build',
-      filename: 'specs.js',
-    },
-  }),
+  })
```

Comment:
I think we needn't export the array anymore. We should also be able to merge the `shared` config directly into this one; it makes no sense factored out anymore.